CDP4CommonView/Diagram/Behaviors/DiagramPortShapeBehavior.cs

```diff
@@ -38,7 +38,15 @@ namespace CDP4CommonView.Diagram.Behaviors
     using ReactiveUI;
 
     /// <summary>
-    /// Allows <see cref="DiagramPortShape"/> to have proper callback 
+    /// The purpose of the <see cref="DiagramPortShapeBehavior"/> is
+    /// <list type="bullet">
```

Comment:
This is weird, i think you missed this.

Reply:
indeed

Reply:
Is it the "is" at the end of line or the list enumerating what the behavior is doing?

Reply:
I dont know, did you do some comment autogen? Just remove it and finish the sentence.

Reply:
I did not do some autogen comment. I thought it would look nice a a list. Anyway I removed the list markup